src/query/terms.js

```diff
@@ -53,6 +53,10 @@ export class Records extends TermBase {
     return new this.constructor(oqe('filter', this.expression, andExpression));
   }
 
+  sort(sortExpressions) {
```

Comment:
Also, I'm not sure about the argument name `sortExpressions`, but I couldn't think of anything better.